batchflow/models/utils.py

```diff
@@ -10,6 +10,8 @@ def unpack_args(args, layer_no, layers_max):
                 arg_value = args[arg][layer_no]
             else:
                 arg_value = args[arg]
+        elif isinstance(args[arg], dict): # for args with dict-like structure, e.g. branch in ResBlock
```

Comment:
This comment really does not help to understand the use case. Also, this function is used in other places, and it may be very sensitive to such changes...

Reply:
Without it, we can't nicely parse something like:
```
config = {
    ...
    'branch/layout' : [...]
    ...
}
```
(A problem is a list in a dict)

Have you got any other **suggestions**? (except hardcode 'branch' key)